setup.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from os.path import exists
+import re
 from setuptools import setup
-import unification
+
+contents = re.findall(
+    r'__version__ = \'([.0-9]+)\'',
+    open('unification/__init__.py', 'r').read())[0]
```

Comment:
This might not work on windows due to the forward slash.

In more recent projects I've just been duplicating the version string in `__init__.py` and `setup.py`.  It's not the most elegant solution but it does work pretty simply.